app/views/ckey2bibframes/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Ckey to Bibframe2 Conversion</h1>
+<div class="btn-group">
+  <%= link_to 'Do another conversion', new_ckey2bibframe_path, class: 'btn btn-md btn-default btn-full' %>
+</div>
+<h3>MarcXML</h3>
+<pre>
+  <%= @ckey2bibframe.marc21_to_xml %>
+</pre>
+<h3>Bibframe2</h3>
+<pre>
+  <%= @ckey2bibframe.marcxml_to_bibframe %>
+</pre>
```

Comment:
It might be, but doing this will take me a lot of time fussing with the css. It is currently laid out the same way as the original display and no one has ever asked for anything different. If it comes up in the future, or if we need to extend the functionality of this in any significant way, we can look into an enhanced display as such.